modules/renter/downloadchunk.go

```diff
@@ -75,7 +82,7 @@ type unfinishedDownloadChunk struct {
 	mu       sync.Mutex
 
 	// Caching related fields
-	chunkCache map[string][]byte
+	chunkCache map[string]cacheData
```

Comment:
my personal preference here would be to make this a `map[string]*cacheData`.

Reply:
oh hmm, actually we do modify the timestamp in place. We modify the timestamp in place when a chunk is accessed in TryCache.

Maybe we should make it a pointer then.